### PR TITLE
feat(#404): Add Params For Instructions in AgentsIT

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
In this PR I added more test packs to `AgentsIT` that demonstrate decompilation process.
What is the most important, now we can specify instruction arguments directly in yaml packs.

Closes: #404.
History:
- **feat(#404): add 'bipush' test that describes the problem**
- **feat(#404): make 'bipush' pack to pass**
- **feat(#404): add hello-world pack test**
- **feat(#404): add 'if' integration test**
- **feat(#404): add one more puzzle**
- **feat(#404): add one more test**

---

### Summary
This PR enhances the testing capabilities of the project by adding more complex test cases involving different agents and instructions.

### Detailed summary
- Added complex test cases with multiple agents and instructions
- Improved parsing of instructions from YAML files
- Updated comments and todos for better code maintenance

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you take a look, please? This PR contains more interesting test cases.

File: src/test/resources/agents/bipush.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+opcodes:
+  - BIPUSH 10
+  - BIPUSH 20
+  - IADD
+agents:
```
Comment (yegor256) [inline src/test/resources/agents/bipush.yaml:5]: @volodya-lombrozo try to change the order of the agents. Maybe not in this test, but in a new one.

**Comment by volodya-lombrozo** [comment]
@yegor256 Thank you for the review! Sure, I added one more [issue](https://github.com/objectionary/opeo-maven-plugin/issues/406) particularly for this task.

**Comment by volodya-lombrozo** [comment]
@rultor merge

**Comment by 0crat** [comment]
@yegor256 Great job on the review! 🎉 You've earned +8 points, which includes bonuses for code contribution and comments. Your running balance is now [+227](https://www.eolang.org/zerocracy/objectionary-vitals.html). Keep up the excellent work, and remember that more detailed reviews with constructive comments can lead to higher rewards!

**Comment by 0crat** [comment]
@volodya-lombrozo Thank you for your contribution! We appreciate your effort in writing 137 hits-of-code. Based on our team's policy, you've earned +29 points: +16 as a base, +13 for the code volume. This aligns with our [quality standards](https://www.yegor256.com/2018/03/06/speed-vs-quality.html). Your current balance is [+235](https://www.eolang.org/zerocracy/objectionary-vitals.html). Keep up the great work and continue delivering valuable contributions!

**Pull request merged**